lib/request-base.js

```diff
@@ -164,6 +164,10 @@ exports.field = function(name, val) {
   if (!this._formData) {
     var FormData = require('form-data'); // browserify compatible. May throw if FormData is not supported natively.
     this._formData = new FormData();
+    this._formData.on('error', function(err) {
```

Comment:
Looks like we don't have a browser test that covers this method. Could you add a shared test that calls 

```js
request.post('/upload')
     .field('foo', 'bar')
     .end(callback);
```

and see if that works? (with `make test-browser-local`)